patch/preconfig/cisco-mdio.patch

```diff
@@ -0,0 +1,40 @@
+From 40effb100908437fa43fe0b8fa9c5802ea1cf162 Mon Sep 17 00:00:00 2001
+From: Madhava Reddy Siddareddygari <msiddare@cisco.com>
+Date: Tue, 9 Mar 2021 08:41:15 -0800
+Subject: [PATCH] Enable MDIO related options
+
+---
+ debian/config/amd64/config | 4 ++++
+ drivers/net/phy/Kconfig    | 4 ++--
+ 2 files changed, 6 insertions(+), 2 deletions(-)
+
+diff --git a/debian/config/amd64/config b/debian/config/amd64/config
+index 4ffe17e12..e3f9ad9e3 100644
+--- a/debian/config/amd64/config
++++ b/debian/config/amd64/config
+@@ -247,3 +247,7 @@ CONFIG_ZONE_DEVICE=y
+ ##
+ CONFIG_LSM_MMAP_MIN_ADDR=65536
+ 
++#
++CONFIG_PHYLIB=m
++CONFIG_OF_MDIO=m
++CONFIG_MDIO_BUS_MUX=m
```

Comment:
please use kconfig-inclusion for all kconfig changes.

Reply:
Hi Guohan,

There are dependencies with other kernel configs and some of the configs are enabled by selecting other configs which are not related to cisco configs.

This PR will make sure only minimum required configs are enabled with this PR

Reply:
Make changes in kconfig-inclusion and add any dependent configurations to kconfig-inclusion